robottelo/constants/__init__.py

```diff
@@ -18,6 +18,8 @@ class Colored(Box):
 # This should be updated after each version branch
 SATELLITE_VERSION = "6.15"
 SATELLITE_OS_VERSION = "8"
+SAT_GA_VERSIONS = ['6.11', '6.12', '6.13']
```

Comment:
Remove this one as you can check against `SAT_NON_GA_VERSIONS` using reverse logic

Reply:
removed